index.html

```diff
@@ -20,27 +22,160 @@
+    </header>
+    <div class="works">
+      <article class="card-1">
+        <img src="./images/tonic1.png" alt="Tonic" class="project-image">
+
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Tonic</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+
+      <article class="card-2">
+        <div class="card-content"></div>
+        <img src="./images/multipost.png" alt="Multi-Post Project" class="project-image">
+
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Multi-Post Project</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+
+
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
 
-    <section class="content">
+      <article class="card-3">
+        <div class="card-content"></div>
+        <img src="./images/tonic2.png" alt="Tonic" class="project-image">
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Tonic</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
```

Comment:
- [ ] Kindly replace `div` with `ul `and `li` tags to list the technologies.

Reply:
Hello! My code already had these technologies as ul and li. Am I missing where should I change this?
```
<ul class="skill-list">
        <li class="list-title">Languages <img src="./resources/downarrow.svg" alt="Down Arrow"></li>
      </ul>
        <ul class="sublist">
          <li class="subitem"> <img src="./resources/jsicon.svg" alt="JavaScript" class="skill-logo"> JavaScript</li>
          <li class="subitem"> <img src="./resources/html5.svg" alt="HTML" class="skill-logo"> HTML</li>
          <li class="subitem"> <img src="./resources/css.svg" alt="CSS" class="skill-logo"> CSS</li>
        </ul>
      <ul class="skill-list">
        <li class="list-title">Frameworks <img src="./resources/rightarrow.svg" alt="Right Arrow"></li>
        <li class="list-title">Skills <img src="./resources/rightarrow.svg" alt="Down Right"></li>
      </ul>
```

Reply:
- HTML file line 112

Reply:
Thanks! I fixed it now, apologies for missing it earlier.